Assets/MixedRealityToolkit/_Core/Services/MixedRealityToolkit.cs

```diff
@@ -391,14 +395,6 @@ private void InitializeInstance()
         // ReSharper disable once StaticMemberInGenericType
         private static bool searchForInstance = true;
 
-        /// <summary>
-        /// Expose an assertion whether the MixedRealityToolkit class is initialized.
-        /// </summary>
-        public static void AssertIsInitialized()
```

Comment:
This cannot be removed and needs to stay.

Reply:
It was called from one location internal to the toolkit object (and it was a simple wrapper around an assert statement. What makes it required? Consistency of message?

Reply:
Because it would be a breaking change.

We need to at least mart it obsolete.

Reply:
Fair enough. I put it back in my recent commit.